lib/knexFunctions.js

```diff
@@ -14,19 +15,133 @@ function createGame(id){
   })
```

Comment:
Hmm. There is a way to make it dynamic, however I want to actually try running the game with it hard coded for simplicities sake. 
If we have a running, working product I can try making it dynamic afterwards : )